resources/var/tmp/resources/init.groovy.d/scripts/init140NORMconfigureLogLevels.groovy

```diff
@@ -1,54 +1,47 @@
-import groovy.json.JsonSlurper;
-import java.util.stream.Collectors;
-import java.util.logging.Level;
-import java.util.logging.Logger;
+package scripts
 
-static Map<String, String> getValuesFromEtcd(String key) {
-    String ip = new File("/etc/ces/node_master").getText("UTF-8").trim();
-    URL url = new URL("http://${ip}:4001/v2/keys/${key}");
+import java.util.logging.Level
+import java.util.logging.Logger
+
+File sourceFile = new File("/var/lib/jenkins/init.groovy.d/lib/EcoSystem.groovy")
+Class groovyClass = new GroovyClassLoader(getClass().getClassLoader()).parseClass(sourceFile)
+ecoSystem = (GroovyObject) groovyClass.getDeclaredConstructor().newInstance()
+
+Map<String, Level> getConfiguredLogLevels() {
     try {
-        def json = new JsonSlurper().parseText(url.text);
-        if (json.node.nodes == null) {
-            println "no valid logging configuration found"
-            return [:]
+        Map<String, Level> loggerLevelMap = new HashMap<>()
+        loggingKeys = ecoSystem.sh("doguctl ls logging").split("\n")
+        for(key in loggingKeys) {
+            logValue = ecoSystem.sh("doguctl config $key")
+            logLevel = getLogLevel(logValue)
+            logKey = key.replace("logging/", "")
+            loggerLevelMap.put(logKey, logLevel)
         }
```

Comment:
why is doguctl here called directly via sh instead of having a wrapper function?

Reply:
Changed